File: hasad/mobile/lib/features/damage_reports/domain/models/damage_report_status.dart
class DamageReportStatus {
  static const String draft = 'Draft';
  static const String techReview = 'TechReview';
  static const String archiveDir = 'ArchiveDir';
  static const String dirManager = 'DirManager';
  static const String minTechReview = 'MinTechReview';
  static const String legalReview = 'LegalReview';
  static const String procReview = 'ProcReview';
  static const String minArchive = 'MinArchive';
  static const String genManager = 'GenManager';
  static const String completed = 'Completed';

  static const List<String> all = [
    draft, techReview, archiveDir, dirManager, minTechReview,
    legalReview, procReview, minArchive, genManager, completed
  ];
}

File: hasad/backend/Hasad.Domain/Constants/DamageReportStatus.cs
namespace Hasad.Domain.Constants;

public static class DamageReportStatus
{
    public const string Draft = "Draft";
    public const string PendingTechnicalVerification = "PendingTechnicalVerification";
    public const string TechReview = "TechReview";
    public const string ArchiveDir = "ArchiveDir";
    public const string DirManager = "DirManager";
    public const string MinTechReview = "MinTechReview";
    public const string LegalReview = "LegalReview";
    public const string ProcReview = "ProcReview";
    public const string MinArchive = "MinArchive";
    public const string GenManager = "GenManager";
    public const string Completed = "Completed";

    public static string[] All() => new[]
    {
        Draft, PendingTechnicalVerification, TechReview, ArchiveDir, DirManager, MinTechReview,
        LegalReview, ProcReview, MinArchive, GenManager, Completed
    };
}
